src/scaffoldmaker/meshtypes/meshtype_3d_tubenetwork1.py

```diff
@@ -112,15 +198,21 @@ def generateBaseMesh(cls, region, options):
         layoutRegion = region.createRegion()
         networkLayout = options["Network layout"]
         networkLayout.generate(layoutRegion)  # ask scaffold to generate to get user-edited parameters
+        layoutAnnotationGroups = networkLayout.getAnnotationGroups()
         networkMesh = networkLayout.getConstructionObject()
 
         tubeNetworkMeshBuilder = TubeNetworkMeshBuilder(
             networkMesh,
             targetElementDensityAlongLongestSegment=options["Target element density along longest segment"],
             defaultElementsCountAround=options["Elements count around"],
             elementsCountThroughWall=options["Elements count through wall"],
-            layoutAnnotationGroups=networkLayout.getAnnotationGroups(),
-            annotationElementsCountsAround=options["Annotation elements counts around"])
+            layoutAnnotationGroups=layoutAnnotationGroups,
+            annotationElementsCountsAround=options["Annotation elements counts around"],
+            defaultElementsCountAcrossMajor=options['Number of elements across major'],
+            elementsCountTransition=options['Number of elements across transition'],
+            annotationElementsCountsAcrossMajor=options["Annotation elements counts across major"],
+            isCore=options["Core"])
+
         tubeNetworkMeshBuilder.build()
         generateData = TubeNetworkMeshGenerateData(
```

Comment:
Make it that `isLinearThroughWall=options["Use linear through wall"] and not options["Core"]`.
Currently fails if Core is on at same time.

Reply:
Added a check in checkCoreParameters function to disable "Use linear through wall" if "Core" option is set to True.